modules/swagger-codegen/src/main/java/com/wordnik/swagger/codegen/examples/ExampleGenerator.java

```diff
@@ -53,12 +65,10 @@ else if(property != null && mediaType.startsWith("application/xml")) {
       }
     }
     else {
```

Comment:
This piece has to be reworked as it renders examples into JSON view regardless of the `contentType` value.